polling_stations/apps/pollingstations/helpers.py

```diff
@@ -0,0 +1,10 @@
+import re
+
+
+def format_postcode_no_space(postcode):
+    return re.sub('[^A-Z0-9]', '', postcode.upper())
+
+
+def format_postcode_with_space(postcode):
+    pc = format_postcode_no_space(postcode)
+    return pc[:-3] + ' ' + pc[-3:]
```

Comment:
I've been meaning to ask this for a while: is the outcode _always_ 3 chars? Seems like an odd bit of consistency!